### PR TITLE
SAKIII-3825 Create HTML/CSS for the Widget Library Developer Information page

### PR DESCRIPTION
**Pull request by bertpareyn**
https://jira.sakaiproject.org/browse/SAKIII-3825

### CODE REVIEW

File: app/views/developer/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <p>Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Cras mattis consectetur purus sit amet fermentum.</p>
+            <p>Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Cras mattis consectetur purus sit amet fermentum.</p>
+            <p>Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Cras mattis consectetur purus sit amet fermentum. Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Cras mattis consectetur purus sit amet fermentum. Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Cras mattis consectetur purus sit amet fermentum.</p>
+            
+            <div id="developer_details">
+                <h2>Details:</h2>
+                <hr class="wl-push wl-hidden">
```
Comment (croby) [inline app/views/developer/index.html.erb]: Can we find a different, maybe CSS-based solution for the issues that these `<hr/>`'s are solving? It seems like a lot of non-semantic markup

File: app/views/developer/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                <hr class="wl-split-line">
+                <div class="developer_details_left">User since:</div><div class="developer_details_right">March 5, 2007</div>
+                <hr class="wl-push wl-hidden">
+                <hr class="wl-split-line">
+                <div class="developer_details_left">No. of Widgets Developed:</div><div class="developer_details_right">7 Sakai Widgets</div>
+                <hr class="wl-push wl-hidden">
+                <hr class="wl-split-line">
+                <div class="developer_details_left">Average rating for their Widgets:</div><div class="developer_details_right">
+                    <div class="wl-rating-container">
```
Comment (croby) [inline app/views/developer/index.html.erb]: Can we use the rating partial here?

File: app/views/developer/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                        <div class="wl-rating-icon"></div>
+                    </div>
+                </div>
+                <hr class="wl-push wl-hidden">
+                <hr class="wl-split-line">
+            </div>
+        </div>
+    </div>
+    <hr class="wl-push wl-hidden">
+    <div id="developer_widgetsbydeveloper" class="wl-inset-shadow-container">
```
Comment (croby) [inline app/views/developer/index.html.erb:36]: Maybe make this a partial?

Comment (bertpareyn) [inline app/views/developer/index.html.erb:36]: are you refering to the developer_widgetsbydeveloper div? 

its contents are already in a partial, do you want the whole div to be in the partial and pass id and title as a variable?

**Comment by bertpareyn** [comment]
Pushed fixes except for the one I have a question about (see above)

**Comment by croby** [comment]
Merged

**Pull request merged**